packages/ui/react-ui-editor/src/extensions/markdown/image.ts

```diff
@@ -63,6 +63,9 @@ const buildDecorations = (from: number, to: number, state: EditorState) => {
         if (urlNode) {
           const hide = state.readOnly || cursor < node.from || cursor > node.to;
           const url = state.sliceDoc(urlNode.from, urlNode.to);
+          if (url.match(/^https?:\/\//) === null) {
```

Comment:
Avoid rendering non-HTTP urls to images.

Reply:
Could you leave a comment here to explain why this is needed?